README.md

```diff
@@ -13,3 +19,18 @@ Panix is the subject of an independent study and the associated final report pub
 
 ### How to Help
 We are constantly looking for people to help contribute to Panix. If you would like to contribute driver code or a new feature, feel free to fork the repository and submit a pull request. If you are a student who would like to get involved, feel free to contact any of the project contributors.
+
+
+### License
+Panix is available under the MIT license. This license may not apply to third party source code used by Panix. Third party source code used is listed below.
+
+See [LICENSE](https://github.com/panix-os/Panix/blob/stable/LICENSE) for details.
+
+## Contributors
+<a href="https://github.com/panix-os/panix/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=panix-os/panix" />
+</a>
+</br>
+
+## Third Party Projects
+* [Limine Bootloader](https://github.com/limine-bootloader/limine)
```

Comment:
It feels a bit strange to say that we use Limine when we haven't merged in that code yet. Maybe we could add this section in that PR once this one gets merged in?

Reply:
Yeah, this is a draft, so I didn't think it would get merged in until appropriate. I was looking for feedback particularly in areas where the README is lacking. I like SkiftOS's design, so I took some inspiration from there.